assets/js/index.js

```diff
@@ -68,42 +68,59 @@ if(main && scrollBtn) {
     observer.observe(main);
 }
 
-docsearch({
-    inputSelector: '.navbar-nav .td-search-input',
-    typesenseCollectionName: 'docsearch',
-    typesenseServerConfig: {
-      nodes: [{
-        host: 'cgnvrk0xwyj9576lp-1.a1.typesense.net',
-        port: '443',
-        protocol: 'https'
-      }],
-      apiKey: 'GHQK6od8KfpvTEh4YpA113gUc2dU5fGR'
-    },
-    typesenseSearchParameters: {
-        group_by: "url_without_anchor",
-        group_limit: 1
-    },
-    // unclear what if anything this achieves.
-    contextualSearch: true
-  });
-docsearch({
-    inputSelector: '.td-sidebar__search .td-search-input',
-    typesenseCollectionName: 'docsearch',
-    typesenseServerConfig: {
-      nodes: [{
-        host: 'cgnvrk0xwyj9576lp-1.a1.typesense.net',
-        port: '443',
-        protocol: 'https'
-      }],
-      apiKey: 'GHQK6od8KfpvTEh4YpA113gUc2dU5fGR'
-    },
-    typesenseSearchParameters: {
-        group_by: "url_without_anchor",
-        group_limit: 1
-    },
-    // unclear what if anything this achieves.
-    contextualSearch: true
-});
+function handleSearch(inputSelector) {
+    const searchConfig = {
+        inputSelector: inputSelector,
+        typesenseCollectionName: 'docsearch',
+        typesenseServerConfig: {
+            nodes: [{
+                host: 'cgnvrk0xwyj9576lp-1.a1.typesense.net',
+                port: '443',
+                protocol: 'https'
+            }],
+            apiKey: 'GHQK6od8KfpvTEh4YpA113gUc2dU5fGR'
+        },
+        typesenseSearchParams: {
+            query_by: 'hierarchy.lvl0,hierarchy.lvl1,url_without_anchor',
+            query_by_weight: '100,50,1',
+            sort_by: "_text_match:desc,item_priority:desc",
+            prioritize_token_position: true,
+            group_by: "url_without_anchor",
+            group_limit: 1
+        },
+        autocompleteOptions: {
+            autoselect: false,
+            debug: false,
+            hint: false
+        }
+    };
+
+    const search = docsearch(searchConfig);
+
+    let opened = false;
+    search.autocomplete.on('autocomplete:shown', (e, a, b, c) => {
+        opened = true;
+    });
+    search.autocomplete.on('autocomplete:closed', () => {
+        opened = false;
+    });
+    let cursorUsed = false;
+    // we can't detect the cursor changing to an empty auto complete entry,
+    // so the only case where getting to search results via Enter will work is
+    // text typed followed by Enter.
+    search.autocomplete.on('autocomplete:cursorchanged', (event, suggestion) => {
+        cursorUsed = true;
+    });
+    search.autocomplete.on('keydown', (e) => {
+        if (opened && !cursorUsed && e.key === 'Enter' && search.input[0].value !== '') {
+            const query = encodeURIComponent(search.input[0].value);
+            window.location = `${window.location.origin}/search?query=${query}`;
+        }
+    });
```

Comment:
commenting this out will disable the search page right? I'd like to disable that until I can take a look at making the pages show up more text

Reply:
doesn't matter if it's otherwise reachable

Reply:
yep I will disable